docs/guide/rest-apis.md

```diff
@@ -141,7 +141,7 @@ Note `pages` and `currentPage`. For performance reasons, not all of the products
 GET /api/v1/product?page=2
 ```
 
-### `GET /api/v1/product/:docId`
```

Comment:
We now have a document property that looks like `docId` (`aposDocId`), so I changed this to be less confusing.

Reply:
`_id` would be exactly correct and then they'd know what it is for sure.